e2e-tests-playwright/blocks/button-maxi/snapshots/button-maxi-content.html

```
@@ -1,3 +1,13 @@
-<!-- wp:maxi-blocks/button-maxi {"maxi-version-current":"2.2.0","maxi-version-origin":"2.2.0","blockStyle":"light","isFirstOnHierarchy":true,"uniqueID":"button-maxi-1se8ef1z-u","customLabel":"Button_1","buttonContent":"Hello","button-padding-top-xxl":"23","button-padding-top-l":"15","button-padding-right-xxl":"55","button-padding-right-l":"36","button-padding-bottom-xxl":"23","button-padding-bottom-l":"15","button-padding-left-xxl":"55","button-padding-left-l":"36","button-padding-sync-xxl":"axis"} -->
-<div class="wp-block-maxi-blocks-button-maxi maxi-block maxi-button-block maxi-light button-maxi-1se8ef1z-u" id="button-maxi-1se8ef1z-u"><button type="button" class="maxi-components-button components-button maxi-button-block__button"><span class="maxi-button-block__content">Hello</span></button></div>
-<!-- /wp:maxi-blocks/button-maxi -->
\ No newline at end of file
+<!-- wp:maxi-blocks/button-maxi {"maxi-version-current":"2.2.1","maxi-version-origin":"2.2.1","blockStyle":"light","isFirstOnHierarchy":true,"uniqueID":"button-maxi-1se8ef1z-u","customLabel":"Button_1","buttonContent":"Hello","button-padding-top-xxl":"23","button-padding-top-l":"15","button-padding-right-xxl":"55","button-padding-right-l":"36","button-padding-bottom-xxl":"23","button-padding-bottom-l":"15","button-padding-left-xxl":"55","button-padding-left-l":"36","button-padding-sync-xxl":"axis"} -->
+<div
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)